web/client/components/styleeditor/Fields.jsx

```diff
@@ -225,9 +229,13 @@ export const fields = {
 
         const [newOptions, setNewOptions] = useState(initOptions(options));
         useEffect(() => {
-            !isEqual(options, newOptions) && setNewOptions(initOptions(options));
+            !multi && !isEqual(options, newOptions) && setNewOptions(initOptions(options));
```

Comment:
is this the missing control that was causing the application freeze?

Reply:
Yes, that's right